intra_process_demo/src/image_pipeline/camera_node.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include <rclcpp/rclcpp.hpp>
```

Comment:
Can we be consistent on the order and how to include the rcl headers? see [this demo](https://github.com/ros2/demos/blob/master/intra_process_demo/src/cyclic_pipeline/cyclic_pipeline.cpp#L20) for comparison